README.md

```diff
@@ -57,15 +57,17 @@ const suborbital = new Suborbital(process.env.SE2_ENV_TOKEN, localUriConfig);
 Then access endpoints on their respective sub-clients:
 
 ```ts
-async function runFunction() {
-  const result = await suborbital.exec.run({
-    environment: "com.acmeco",
-    userId: "1234",
-    namespace: "default",
-    fnName: "foo",
-    version: "v1.0.0",
-  });
-
-  console.log("Function output:", result);
+async function run() {
+  const result = await suborbital.exec.run(
+    {
+      environment: "com.acmeco",
+      userId: "1234",
+      namespace: "default",
+      fnName: "hello",
```

Comment:
Should this flip and be a breaking change now or no @jagger27?

Reply:
Yes